Mathlib/Analysis/InnerProductSpace/JointEigenspace.lean

```diff
@@ -8,18 +8,27 @@ import Mathlib.Analysis.InnerProductSpace.Spectrum
 import Mathlib.Analysis.InnerProductSpace.Projection
 import Mathlib.Order.CompleteLattice
 import Mathlib.LinearAlgebra.Eigenspace.Basic
+import Mathlib.LinearAlgebra.Eigenspace.Pi
+import Mathlib.LinearAlgebra.Eigenspace.Semisimple
+import Mathlib.Analysis.InnerProductSpace.Semisimple
```

Comment:
Can you run `#min_imports` at the bottom of the file to see which of these can be removed? I suspect that there are redundant imports.

Reply:
Sure. I'm surprised that the linter didn't catch this...